### DISCUSSION TITLE
Running Tests Locally

### DISCUSSION THREAD

**Discussion by aaronmgdr** [wevm/viem#716] [Question] [answered]
Im trying to run the test locally and Im running into an issue. 

I have forge/ foundry installed

when i run `pnpm test` about half the test fail. 

when I run an isolated test file, such as signTransaction.test.ts The test all pass but the got an overall failure on the test suite due to 'fetch not found' error 

wondering if i missed a setup stage.

**Comment by tmm** [comment, marked as answer]
Make sure you are using Node 18+ (https://viem.sh/docs/compatibility.html)
